frontend/android/build.gradle

```diff
@@ -25,6 +25,7 @@ android {
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         multiDexEnabled true
+        resConfigs "en", "ja"
```

Comment:
`en` is needless because `en` resource does not exist in this app.
English materials are in the default resources so `resConfigs "ja"` is enough as for this app. 👍 

To check the difference, it would be nice if you can compare between each result of `aapt dump badging $apk_path`.